src/components/VideoSuggestion/styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  padding-bottom: 5px;
+`;
```

Comment:
I like how this one works, you can improve the UX a lot by adding one line: 
```suggestion
const Container = styled.div`
  width: 100%;
  display: flex;
  padding-bottom: 5px;
  cursor: pointer;
`;
```
With `cursor: pointer;` you let the user know that if the card is clicked something will happen, just a little detail

Reply:
Awesome! I also added this property to my `VideoCard`. Great suggestion!